### PR TITLE
Feature/us 45 UI coin change page

### PR DESCRIPTION
**Pull request by victor0villca**
## What?
I created the UI page of the Coin Change Exercise in Next.js

## How?
This was done with different components that i created in order to do the UI.

## Why?
To be able to have a page the user can interact in order to do the coin change process.

## Mockup
https://www.figma.com/file/yYcgC1y2krnbnE7p1KnoBW/Untitled?type=design&node-id=0-1&mode=design

#Author 
@victor0villca

### CODE REVIEW

File: .idea/.gitignore
Comment (CatCoder36) [inline .idea/.gitignore:1]: We no longer track the .idea folder

**Comment by zrock-dev** [comment]
Please re-branch from develop and cherry-pick your branch commits.

**Comment by CatCoder36** [comment]
image

The display of the coins should be modified by making the screen smaller.

File: .idea/.gitignore
Comment (victor0villca) [reply .idea/.gitignore:1]: Already solved.

**Review by zrock-dev — COMMENTED** [review]
Since your changes are from the frontend, I think you should request reviews from developers that are working on it like @luiggymamani

**Review by CatCoder36 — APPROVED** [review]
Good Job

**Review by zrock-dev — APPROVED** [review]

**Review by zrock-dev — APPROVED** [review]

**Pull request merged**